src/vat/scoreboard.js

```diff
@@ -109,40 +111,41 @@ export function makeScoreboard(quorumTest, logConflict) {
  * joining, old followers leaving, change of leader, change of
  * threshold, or any other change to the quorum rules.
  */
-function makeConsensusLeader(decidedQs) {
-  // map from compositeID to scoreboard
-  const scoreboards = new Map();
 
-  function deliverMsg(compositeID, msg) {
-    for (const decidedQP of decidedQs) {
-      decidedQP.e.deliverMsg(compositeID, msg);
-    }
-  }
+// function makeConsensusLeader(decidedQs) {
+//   // map from compositeID to scoreboard
+//   const scoreboards = new Map();
 
-  return harden({
-    acceptProtoMsg(compositeID, componentID, seqNum, msgID, msg) {
-      let scoreboard = scoreboards.get(compositeID);
-      if (!scoreboard) {
-        scoreboard = makeScoreboard(getQuorumTest(compositeID));
-        scoreboards.set(compositeID, scoreboard);
-      }
-      if (scoreboard.acceptProtoMsg(componentID, seqNum, msgID, msg)) {
-        // Currently, because we're exhaustive here, we know there's
-        // nothing ready on the other scoreboards. Instead, we may
-        // want to not be exhaustive in order to do a fairer
-        // interleaving of messages from different compositeID
-        // sources.
-        while (true) {
-          const msg = scoreboard.getNext();
-          if (!msg) {
-            return;
-          }
-          deliverMsg(compositeID, msg);
-        }
-      }
-    },
-  });
-}
+//   function deliverMsg(compositeID, msg) {
+//     for (const decidedQP of decidedQs) {
+//       decidedQP.e.deliverMsg(compositeID, msg);
+//     }
+//   }
+
+//   return harden({
+//     acceptProtoMsg(compositeID, componentID, seqNum, msgID, msg) {
+//       let scoreboard = scoreboards.get(compositeID);
+//       if (!scoreboard) {
+//         scoreboard = makeScoreboard(getQuorumTest(compositeID));
+//         scoreboards.set(compositeID, scoreboard);
+//       }
+//       if (scoreboard.acceptProtoMsg(componentID, seqNum, msgID, msg)) {
+//         // Currently, because we're exhaustive here, we know there's
+//         // nothing ready on the other scoreboards. Instead, we may
+//         // want to not be exhaustive in order to do a fairer
+//         // interleaving of messages from different compositeID
+//         // sources.
+//         while (true) {
+//           const nextMsg = scoreboard.getNext();
+//           if (!nextMsg) {
+//             return;
+//           }
+//           deliverMsg(compositeID, nextMsg);
+//         }
+//       }
+//     },
+//   });
+// }
```

Comment:
And we aren't using `makeConsensusFollower` anywhere either, we could probably comment it out the same way, but in a separate commit since that's a less-obviously-trivial change.

Reply:
I think makeConsensusFollower is already commented out by Mark